ts/create-kpt-functions/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "create-kpt-functions",
-  "version": "0.17.0",
+  "version": "0.18.0",
```

Comment:
So make sure this release is validated. Given you haven't created an `rc` release, anyone running `npm init kpt-functions` will be affected when releasing this. 

In other words, you're skipping step 4 and going to step 8:

https://github.com/GoogleContainerTools/kpt-functions-sdk/blob/master/CONTRIBUTING.md#create-kpt-functions

If there's an issue, then need to roll forward a fix.

Reply:
On second thought, I recommend creating an rc release and testing it locally given it's been a year since last release.

Reply:
Updated